.github/workflows/test.yml

```diff
@@ -58,7 +58,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
I would personally prefer testing every minor version. Especially for a library like this that tightly integrates with CPython, there may be differences in minor versions we want to test. We can drop support for 3.9 or other versions if you're concerned about the total number of tests.